ECMAScript/react-native/README.md

```diff
@@ -0,0 +1,24 @@
+# Quiltt Connector React Native SDK
+
+```typescript
+import { useState } from 'react'
+import { QuilttConnector } from '@quiltt/react-native'
+
+export const App = () => {
+  const [connectionId, setConnectionId] = useState<string>()
+  const oAuthRedirectUrl = "quilttexample://open.reactnative.app"
```

Comment:
Would it be possible to automatically detect/generate this in the future?

Reply:
Generate readme or the entire sdk?

Reply:
Do we have anyway to generate other packages readme at the moment?